packages/@orchidui-vue/src/MediaAndIcons/Icon/OcIcon.vue

```diff
@@ -29,8 +29,10 @@ const props = defineProps({
 });
 const iconRef = ref(null);
 fetch(`${props.path}/${props.name}.svg`)
-  .then((r) => r.text())
+  .then((r) => (r.status === 200) ?r.text() : '')
```

Comment:
can we add error handling here to indicate if the SVG file itself is present or not. I'm thinking it would throw an error if the file is not available.